### PR TITLE
Add commit name check 

### LINKED ISSUE #14: Add commit name check via GitHub Actions
The suggested commit name design here:
`<branch-id>: <commit-name>`

So issue #1 will have commits like this (on branch _1-issue-fix_ for example):

```
1: Chore main.py
1: Fix script.sh
1: Add note to README.md 
```

### PR DESCRIPTION
**Pull request by SSSKrut**
Close #14

### CODE REVIEW

**Comment by butvinm** [comment]
@SSSKrut Could you check this warning: https://github.com/butvinm/nobutt/pull/15/checks#step:3:6

File: Makefile [hunk 1]
```diff
@@ -3,10 +3,16 @@ check_prerequisites:
 	@echo "Python version: $(shell python --version)"
 	@echo "Poetry version: $(shell poetry --version)"
 	@echo "Npm version: $(shell npm --version)"
+	@sh ./scripts/check_hooks.sh
```
Comment (butvinm) [inline Makefile:9]: Developer has to run `make install` and hooks would be set anyway, do we really need this check in the prerequisites?

File: .github/workflows/commit-message-check.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get pushed commits
+        id: get_commits
+        run: echo "::set-output name=commits::$(git log ${{ github.event.before }}..${{ github.event.after }} --pretty=format:'%s')"
```
Comment (butvinm) [inline .github/workflows/commit-message-check.yml]: This line causes a warning: https://github.com/butvinm/nobutt/pull/15/checks#step:3:6

**Review by butvinm — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (SSSKrut) [reply Makefile:9]: `check_prerequisites` writes the versions of the packages used, I think it's a good idea to inform the contributor at this stage that something not configured. Anyway, it must be done. Here, the easy way to do it

File: .github/workflows/commit-message-check.yml (hunk 2 above)
Comment (SSSKrut) [reply .github/workflows/commit-message-check.yml]: Uh oh, sure

**Comment by SSSKrut** [comment]
@butvinm Check out the changes

**Review by butvinm — APPROVED** [review]

**Pull request merged**